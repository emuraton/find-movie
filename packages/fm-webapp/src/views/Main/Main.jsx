import React from 'react'
import { BrowserRouter as Router, Route } from 'react-router-dom'
import { Header } from 'fm-components'
import SearchView from '../SearchView/SearchView'
import MovieDetails from '../MovieDetails/MovieDetails'

export default () => {
  return (
    <Router>
      <div>
        <Route path="/" component={Header}/>
        <Route exact path="/" component={SearchView}/>
        <Route path="/movie" component={MovieDetails}/>
      </div>
    </Router>

  )
}
