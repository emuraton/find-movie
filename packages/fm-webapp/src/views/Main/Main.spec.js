import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

jest.mock('fm-components', () => ({
    Header: 'HeaderMocked'
}))

import Main from './Main'

describe('<Main />', () => {
  describe('@renders', () => {
    it('in default state', () => {
      const wrapper = shallow(<Main />)
      expect(toJson(wrapper))
        .toMatchSnapshot()
    })
  })
})
