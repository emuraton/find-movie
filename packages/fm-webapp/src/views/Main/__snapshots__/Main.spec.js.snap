// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> @renders in default state 1`] = `
<BrowserRouter>
  <div>
    <Route
      component="HeaderMocked"
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/movie"
    />
  </div>
</BrowserRouter>
`;
